CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ macro(build_mimick)
       list(APPEND cmake_configure_args -G ${CMAKE_GENERATOR})
     endif()
     if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "^(x86_|x86-|AMD|amd|x)64$")
-      list(APPEND cmake_configure_args -A x64)
+      list(APPEND cmake_configure_args "-DMIMICK_TARGET_ARCH=x64")
     endif()
```

Comment:
I know you didn't add this, but it almost seems like this is the wrong place to be doing this.  That is, this is the vendor package; shouldn't the actual package be the one determining what target architecture it should use?  I guess we may have to pass a variable along (in the case of cross-compiling), but otherwise it seems like this detection shouldn't even be here.  Thoughts?